### PR TITLE
Feature/multiple archive support

### PR DESCRIPTION
**Pull request by SecondeJK**
This documents the ability to pass in a Multi Archive tag into the methods that start an Archive or a Broadcast.

## Description
This is mostly documentation.

## Motivation and Context
Missing feature that is live.

## How Has This Been Tested?
Two new tests added to make sure that adding custom options do not throw exceptions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by jeffswartz — CHANGES_REQUESTED** [review]
The Archive class should include a `multiArchiveTag` property. And the Broadcast class should include a `multiBroadcastTag` property.

File: src/OpenTok/Broadcast.php [hunk 1]
```diff
@@ -35,6 +35,10 @@
 * @property boolean $isStopped
 * Whether the broadcast is stopped (true) or in progress (false).
 *
+* @property string $multiArchiveTag
+* Whether Multiple Archive is switched on, which will be a unique string for each simultaneous broadcast of an ongoing session.
+* See https://tokbox.com/developer/guides/archiving/#simultaneous-archives for more information.
+*
```
Comment (jeffswartz) [inline src/OpenTok/Broadcast.php:41]: This should be `multiBroadcastTag`

File: src/OpenTok/Broadcast.php [hunk 2]
```diff
@@ -69,6 +73,8 @@ class Broadcast
     private $isLowLatency;
     /** @ignore */
     private $isDvr;
+    /** @ignore */
+    private $multiArchiveTag;
```
Comment (jeffswartz) [inline src/OpenTok/Broadcast.php:77]: Broadcast, not archive. (Same comment for the rest of this file.)

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: src/OpenTok/Broadcast.php [hunk 3]
```diff
@@ -35,6 +35,10 @@
 * @property boolean $isStopped
 * Whether the broadcast is stopped (true) or in progress (false).
 *
+* @property string $multiBroadcastTag
+* Whether Multiple Broadcast is switched on, which will be a unique string for each simultaneous broadcast of an ongoing session.
+* See https://tokbox.com/developer/guides/archiving/#simultaneous-archives for more information.
```
Comment (jeffswartz) [inline src/OpenTok/Broadcast.php:40]: This should be  https://tokbox.com/developer/guides/broadcast/live-streaming/#simultaneous-broadcasts.

**Review by jeffswartz — CHANGES_REQUESTED** [review]

**Review by pardel — APPROVED** [review]

**Pull request merged**